racetrack_commons/racetrack_commons/api/asgi/asgi_server.py

```diff
@@ -26,20 +28,44 @@
     'GET /metrics/ 200',
 }
 
-UVICORN_DEBUG_LOGS = False
+UVICORN_ERROR_LOGS = True
```

Comment:
This looks like a leftover from testing.

Reply:
No, I left that on purpose. It turns out that uvicorn sometimes spits out something useful, especially when it receives a signal and shuts down. Plus, it's not a good idea to keep the potential errors silent.